resources/assets/uad_lists.json

```
@@ -37628,6 +37670,18 @@
     "id": "com.motorola.thermalservice",
     "list": "Oem",
     "description": "Motorola Thermal Service\nTurns off phone when it's too hot.",
+    "id": "com.oplus.apprecover",
+    "list": "Oem",
+    "description": "Recover system apps.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
Don't you think this should be `Advanced` instead? Because this app has more permissions than other regular apps from the Play Store in order to make complete back-ups. I could be wrong though.

---

uninstallable system app recovery to advanced? it may be ok to set advanced when no trackers, ads, but never seen it on other phones

---

I'm not sure if it has no trackers or ads, but I don't think it's "upright pointless".